2002-07-17 Dmitry V. Levin <ldv@altlinux.org>
   * bugfix: restored fields order in struct Gpm_Event to avoid
     ABI change.

--- gpm-1.20.1rc1.orig/src/headers/gpm.h	2002-07-17 18:02:15 +0400
+++ gpm-1.20.1rc1/src/headers/gpm.h	2002-07-17 18:02:47 +0400
@@ -123,10 +123,10 @@
   unsigned char buttons, modifiers;  /* try to be a multiple of 4 */
   unsigned short vc;
   short dx, dy, x, y;
-  short wdx, wdy;
   enum Gpm_Etype type;
   int clicks;
   enum Gpm_Margin margin;
+  short wdx, wdy;
 }              Gpm_Event;
 
 /*....................................... The handling function */
